### PR TITLE
New version: Ryujinx.Ryujinx.Ava version 1.1.1106

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133888)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Ryujinx.Ryujinx.Ava
+PackageVersion: 1.1.1106
+PackageLocale: en-US
+Publisher: Ryujinx
+PublisherUrl: https://ryujinx.org
```
Comment (agatemosu) [inline manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://ryujinx.org/
```

File: manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Ryujinx.Ryujinx.Ava
+PackageVersion: 1.1.1106
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: publish\Ryujinx.Ava.exe
+  PortableCommandAlias: Ryujinx.Ava
+ReleaseDate: 2024-01-13
```
Comment (agatemosu) [inline manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.installer.yaml:11]:
```suggestion
```

File: manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Ryujinx/release-channel-master/releases/download/1.1.1106/ryujinx-1.1.1106-win_x64.zip
```
Comment (agatemosu) [inline manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.installer.yaml:14]:
```suggestion
  InstallerUrl: https://github.com/Ryujinx/release-channel-master/releases/download/1.1.1106/test-ava-ryujinx-1.1.1106-win_x64.zip
```

File: manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  InstallerSha256: B719A0C2A492787D27E12F1E677558305139DDC8563BF5B7054E9C89E01606F5
```
Comment (agatemosu) [inline manifests/r/Ryujinx/Ryujinx/Ava/1.1.1106/Ryujinx.Ryujinx.Ava.installer.yaml:15]:
```suggestion
  InstallerSha256: FBA07A47A9F45ED3AF646E87A0F942B46A9079323B673B897311E677EB2C6A50
```

**Comment by agatemosu** [comment]
There is a new version: https://github.com/microsoft/winget-pkgs/pull/133920

**Comment by Exorcism0666** [comment]
@agatemosu Older versions can still be added to Winget if the download link is valid!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**